src/main/java/de/uhd/ifi/se/decision/management/jira/persistence/ActiveObjectStrategy.java

```diff
@@ -270,7 +280,7 @@ public Long doInTransaction() {
 				if (sourceElements.length == 1) {
 					sourceElement = sourceElements[0];
 				}
-				
```

Comment:
Code readability improvement:
We could already terminate the execution here with an else block, be more specific than at the line 292: `
 				if (sourceElement == null || destinationElement == null) {
 					LOGGER.error("One of the elements to be linked does not exist.");
 					return (long) 0;
				}`